src/Blocks/Blocks.php

```diff
@@ -2,6 +2,7 @@
 
 namespace AstraticBlocks\Blocks;
 
+use AstraticBlocks\Blocks\Base;
```

Comment:
why is this here?
looks like the class is not used here 🤔

Reply:
This is a small improvement for the code from previous work here: https://github.com/wpserved/astratic-blocks/pull/4/files#diff-16373bb60f50bc0484b2beb32047fc2945e0493d2cc407d556f0015129aefff0. I've talked with @coditive-przemek-h about it and he said there is no need to create new branch only to add this one line.